src/dex/balancer-v2/balancer-v2-math.ts

```diff
@@ -164,12 +183,12 @@ class LogExpMath {
   // two numbers, and multiply by ONE when dividing them.
 
   // All arguments and return values are 18 decimal fixed point numbers.
-  static ONE_18: bigint = BigInt('1000000000000000000');
+  static ONE_18: bigint = BI_POWS[18];
```

Comment:
Couldn't we delete the line and replace all occurrences of `ONE_18` with `BI_POWS[18]`? Same goes with the next two lines for `ONE_20` and `ONE_36`.
Just to reduce confusion and avoid mistakes

Reply:
Fixed